ENIGMAsystem/SHELL/Universal_System/Extensions/ttf/find-win32.cpp

```diff
@@ -0,0 +1,29 @@
+#include "find.h"
+
+#include <windows.h>
+
+namespace enigma {
+
+std::string font_lookup(std::string name, bool bold, bool italic, unsigned char** buffer, size_t &buffer_size) {
+  HDC hDC = GetDC(NULL); // just grab the "common" device context of the entire desktop
+
+  LOGFONT requestedFont = { };
+  requestedFont.lfWeight = bold ? FW_BOLD : FW_NORMAL;
+  requestedFont.lfItalic = italic;
+  requestedFont.lfCharSet = DEFAULT_CHARSET;
+  strncpy(requestedFont.lfFaceName, name.c_str(), name.length() < LF_FACESIZE ? name.length() + 1 : (LF_FACESIZE - 1));
+  if (name.length() >= LF_FACESIZE) requestedFont.lfFaceName[LF_FACESIZE - 1] = '\0';

```

Comment:
@JoshDreamland please check my string copy logic here when you get a chance. Thanks.